components/toast/Toast.js

```diff
@@ -1,34 +1,27 @@
 import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
-import Transition from 'react-transition-group/Transition';
 import cx from 'classnames';
 import { IconButton, LinkButton } from '../button';
 import { TextBody } from '../typography';
 import LoadingSpinner from '../loadingSpinner';
-import { createPortal } from 'react-dom';
 import { IconCloseMediumOutline } from '@teamleader/ui-icons';
 import theme from './theme.css';
 
 class Toast extends PureComponent {
-  toastRoot = document.createElement('div');
-
   componentDidMount() {
-    document.body.appendChild(this.toastRoot);
-
-    if (this.props.active && this.props.timeout) {
+    if (this.props.timeout) {
       this.scheduleTimeout(this.props);
     }
   }
 
   componentWillReceiveProps(nextProps) {
```

Comment:
PP: since code was already there, but componentWillRecieveProps is deprecated. Perhaps use componentDidUpdate. Or is this part of a later PR?